crates/sui-indexer-alt-e2e-tests/tests/graphql/events/filter/checkpoint.move

```diff
@@ -85,7 +85,7 @@ module test::events_test {
       ...E
     }
   }
-  eventsBeforeCheckpoint4: events(last: 50, filter: {beforeCheckpoint: 3}) {
```

Comment:
Had a typo on this before, it should show events up to checkpoint 4, the snapshot should include 3 more events from cp 3